src/js/control-bar/time-controls/remaining-time-display.js

```diff
@@ -36,19 +38,16 @@ class RemainingTimeDisplay extends TimeDisplay {
   }
 
   /**
-   * The remaining time display prefixes numbers with a "minus" character.
-   *
-   * @param  {number} time
-   *         A numeric time, in seconds.
+   * Create the `Component`'s DOM element with the "minus" characted prepend to the time
    *
-   * @return {string}
-   *         A formatted time
-   *
-   * @private
+   * @return {Element}
+   *         The element that was created.
    */
-  formatTime_(time) {
-    // TODO: The "-" should be decorative, and not announced by a screen reader
-    return '-' + super.formatTime_(time);
+  createEl(plainName) {
+    const el = super.createEl();
```

Comment:
Since this method accepts a `plainName`, it should pass it upwards as `super.createEl(plainName);`.

Reply:
looking at the super component, it seems like `plainName` isn't used anywhere. We can still make the change if you think it's best to have it in (or remove `plainName` from the TimeDisplay component).

Reply:
I went ahead and removed `plainName` from the remaining time and time display components.